gatherers/theme-color.js

```diff
@@ -26,7 +27,10 @@ class ThemeColor extends Gather {
     return driver.querySelector('head meta[name="theme-color"]')
       .then(node => node && node.getAttribute('content'))
       .then(themeColorMeta => {
-        return {themeColorMeta};
+        if (themeColorMeta && validColor(themeColorMeta)) {
```

Comment:
so setting up a test for this made it clear it was a bit weird to have in the gather phase. I've pushed it into the audit itself, and added tests for it there :)